runtime/service.go

```diff
@@ -68,12 +68,18 @@ func init() {
 }
 
 const (
-	defaultVsockPort        = 10789
-	minVsockIOPort          = uint32(11000)
-	firecrackerStartTimeout = 5 * time.Second
-	defaultStopVMTimeout    = 5 * time.Second
-	defaultShutdownTimeout  = 5 * time.Second
-	jailerStopTimeout       = 3 * time.Second
+	defaultVsockPort = 10789
+	minVsockIOPort   = uint32(11000)
+
+	// vmReadyTimeout is used to control the time all requests wait a Go channel (vmReady) before calling
+	// Firecracker's API server. The channel is closed once the VM starts.
+	vmReadyTimeout = 5 * time.Second
+
+	defaultCreateVMTimeout = 20 * time.Second
+	defaultStopVMTimeout   = 5 * time.Second
+	defaultShutdownTimeout = 5 * time.Second
+
+	jailerStopTimeout = 3 * time.Second
```

Comment:
Does it make sense to make the rest of these configurable also?

Reply:
- StopVM's timeout is configurable and defaultStopVMTimeout is the default value. We don't have to make it configurable.
- Shutdown() is only exposed to containerd. We cannot have a request-level timeout. We could have a configuration variable on firecracker-runtime.json, but I'd like to keep it as is, to keep our configuration simpleler.
- jailerStopTimeout is not doing compute-intensive stuff, compared to CreateVM. 3 seconds may be enough for everyone.

So, right now I don't think we need to expose these knobs to clients.